python/cudf/cudf/io/dlpack.py

```diff
@@ -68,9 +68,6 @@ def to_dlpack(cudf_obj):
     cuDF to_dlpack() produces column-major (Fortran order) output. If the
     output tensor needs to be row major, transpose the output of this function.
     """
-    if len(cudf_obj) == 0:
-        raise ValueError("Cannot create DLPack tensor of 0 size")
-
```

Comment:
No longer true, we can, as long as a DataFrame has at least one column it's ok. `libdlpack.to_dlpack` raises `ValueError: PyCapsule_New called with null pointer` if the dataframe is completely empty. I can catch this and produce a more user-friendly error message if we prefer.